src/librustdoc/formats/cache.rs

```diff
@@ -293,7 +293,8 @@ impl<'a, 'tcx> DocFolder for CacheBuilder<'a, 'tcx> {
                                 ItemType::Trait
                                 | ItemType::Struct
                                 | ItemType::Union
-                                | ItemType::Enum,
+                                | ItemType::Enum
+                                | ItemType::Typedef,
```

Comment:
This looks correct but I don't understand why this tweak only applies to Trait, Struct, Union, Enum, and Typedef. Why not other ItemTypes like Function, Macro, Constant, etc?